chains/manager.go

```diff
@@ -271,13 +268,13 @@ type manager struct {
 }
 
 // New returns a new Manager
-func New(config *ManagerConfig) Manager {
+func New(config *ManagerConfig, subnets *Subnets) Manager {
```

Comment:
Should we just pass in `subnets` as part of the `ManagerConfig` in accordance with all of the other parameters?

It feels a bit weird that `subnets` would be the one exception here

Reply:
I think long-term I would like to see us only store actual config values in `*Config` structs but I think it makes sense to put it in the config as per your comment for consistency for the time being